### DISCUSSION TITLE
Support for lemon squeezy

### DISCUSSION THREAD

**Discussion by qiweiii** [nextify-limited/saasfly#8] [Q&A]
Many of us cannot register for stripe or stripe account is frozen. It would be greate if add other payment solutions like lemon squeezy. Anyway, really awesome project, thanks for making this public!

**Comment by nextify2025** [comment]
Thank you for your interest in Saasfly Adding new features is relatively easy, but balancing functionality and maintainability is a significant challenge for us. As a brand new open-source project, we highly value input from the community and will gradually refine our roadmap based on the feedback we receive.
